docs/user-guide/monitoring.md

```diff
@@ -44,7 +44,7 @@ kubectl get svc -n ingress-nginx
 NAME                   TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                                      AGE
 default-http-backend   ClusterIP   10.103.59.201   <none>        80/TCP                                       3d
 ingress-nginx          NodePort    10.97.44.72     <none>        80:30100/TCP,443:30154/TCP,10254:32049/TCP   5h
-prometheus             NodePort    10.98.233.86    <none>        9090:32630/TCP                               1m
+prometheus-service     NodePort    10.98.233.86    <none>        9090:32630/TCP                               1m
```

Comment:
I'd change the name in the manifest, _service_ is redundant.

Reply:
from the doc, the service was created with https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/monitoring/prometheus.yaml.

Reply:
That's exactly what I meant, please change the name in the manifest, not in the docs.

Reply:
so should we also need to update the `prometheus-server` to `prometheus` as well?

Reply:
No this is fine. What I want to avoid is the use of Kubernetes object _types_ in Kubernetes object _names_ (e.g. `myapp-pod`, `myapp-deployment`, `myapp-service` are generally considered as bad names).

Reply:
thanks, I see.